# Request
string floor_name
---
# Response
bool success
string message